src/VisualStudio/Core/Def/KeybindingReset/KeybindingResetDetector.cs

```diff
@@ -362,7 +360,7 @@ private void RestoreVsKeybindings()
 
     private void OpenExtensionsHyperlink()
     {
-        ThisCanBeCalledOnAnyThread();
+        // ThisCanBeCalledOnAnyThread();
```

Comment:
Switched these to be docs instead.

Reply:
Some instance of `ThisCanBeCalledOnAnyThread` were removed, and some were changed to comments. Either way works for me, but it would be good to be consistent.